src/util/command_line_handling.cpp

```diff
@@ -667,8 +667,7 @@ namespace hpx { namespace util
             }
         }
 
-        if ((vm.count("hpx:connect") || mode_ == hpx::runtime_mode_connect) &&
-            hpx_host == "127.0.0.1")
+        if (hpx_host == "127.0.0.1" || hpx_host == "localhost")
```

Comment:
Is this code here still needed? I think that's what's causing the hang on circleci.